stable/traefik/templates/service.yaml

```diff
@@ -14,5 +14,7 @@ spec:
   ports:
   - port: 80
     name: http
+{{- if .Values.ssl.enabled }}
```

Comment:
Can we take this guard out of this PR? I agree it probably belongs, but it's unrelated to everything else this PR is trying to accomplish. Smaller, more tightly-scoped PRs are more tractable.